_posts/2025-04-28-visualizar-simulaciones-advance.md

```
@@ -0,0 +1,56 @@
+### Visualizar simulaciones
+
+En Advance ahora es mucho más fácil ver las simulaciones de tus clientes, estas están en un solo lugar con toda la información que necesitas en una sola pantalla optimizando al máximo tu tiempo.
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/
+img/simulaciones_menu.jpg" alt="menusimulaciones">
+</p>
+
+Accediendo a la simulación que deseas revisar podrás ver la información de tu cliente.
+
+<p align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)